README.md

```diff
@@ -11,6 +11,23 @@ part of the array each recursive call considers.
 
 ## Runtime Analysis
 
-Analyse the time complexity of your implementation and give a $\Theta$ bound for
+Analyze the time complexity of your implementation and give a $\Theta$ bound for
 its worst-case runtime. Add your answer, including your reasoning, to this
 markdown file.
+
+Complexity Analysis: 
+
+Looking back at my code for the 100th time now. 
+
+Within my code in the for(let length = 1; length < n; length *=2). This is considered the outer loop in this code and with iteration, it does it will double the length of the array and when the length reaches the length of the array that has been inserted into the function it will stop. This will produce _O_(log n). 
+
+Then the inner loop that I have for(let left = 0; left < n - 1; left += 2 * length) { let mid = Math.min(left + length - 1, n - 1); let right = Math.min(left + 2 * length - 1, n-1);This will divide the arrays into subarrays and organize them into the sorted array that is needed in the end. Since we have the left and right doing n iterations because of the swapping within the arrays. This would be _O_(n^2).
```

Comment:
Not following you -- you have a loop that runs log n times and get $n^2$ in the end?